website/content/en/highlights/2022-08-16-0-24-0-upgrade-guide.md

```diff
@@ -81,3 +83,15 @@ This has two potential implications that you should consider.
    the `native` codec) make sure you upgrade the consumers first followed by the
    producers to ensure newer versions of Vector aren't sending data to older
    versions, which may not be able to be read.
+
+#### `codec` field on sink `encoding` must be specified explicitly {#sink-encoding-codec}
+
+Setting the `encoding` value on a sink by a string has been deprecated in
+`0.23.0` and will be removed in this release. To migrate have a look at the
+[upgrade guide for deprecated shorthand values for `encoding` options](https://vector.dev/highlights/2022-07-07-0-23-0-upgrade-guide/#deprecated-encoding-shorthand).
```

Comment:
Generally for links we avoid including the domain so that branch deploys work correctly. Same for the below link.
```suggestion
[upgrade guide for deprecated shorthand values for `encoding` options](/highlights/2022-07-07-0-23-0-upgrade-guide/#deprecated-encoding-shorthand).
```